scripts/docker/test.bash

```diff
@@ -3,6 +3,17 @@
 set -eu
 set -o pipefail
 
+if [ -f /sys/fs/cgroup/cgroup.controllers ]; then
+    echo "Setting up controllers for cgroup v2"
+    # move the processes from the root group to the /init group,
+    # otherwise writing subtree_control fails with EBUSY
+    # An error during moving non-existent process (i.e., "cat") is ignored.
+    mkdir -p /sys/fs/cgroup/init
+    xargs -rn1 < /sys/fs/cgroup/cgroup.procs > /sys/fs/cgroup/init/cgroup.procs || :
```

Comment:
I'm not familiar with the `|| :` syntax? What does it do?

Reply:
This will move the contents of /sys/fs/cgroup/cgroup.procs to /sys/fs/cgroup/init/cgroup.procs